react/src/lib/components/DeckGLMap/layers/wells/wellsLayer.stories.tsx

```diff
@@ -105,16 +110,16 @@ CallbackDashedWells.args = {
         {
             "@@type": "WellsLayer",
             data: "@@#resources.wellsData",
-            dashed: callback,
+            style: callback,
```

Comment:
Maybe it would be better to have each style attributes (dashed, trajectory color, head color, etc) as optionally respective callbacks. This way you can implement only one attribute as callback and the other as values more easily and also filter on wells with different respective criteria.

Reply:
Nice suggestion! I will do the change.